src/R/Components/Impl/ConnectionManager/Implementation/View/ConnectionManagerControl.xaml

```diff
@@ -8,6 +8,10 @@
              xmlns:designTime="clr-namespace:Microsoft.R.Components.ConnectionManager.Implementation.View.DesignTime"
              xmlns:viewModel="clr-namespace:Microsoft.R.Components.ConnectionManager.ViewModel"
              xmlns:controls="clr-namespace:Microsoft.R.Wpf.Controls;assembly=Microsoft.R.Wpf"
+             xmlns:imaging="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.Imaging"
```

Comment:
Since we are not building completely separate app, not referencing package-level things like *Interop and *Shell should be fine since tests will work without VS. If we were to build a separate app, we could look deeper.